en/core-libraries/httpclient.rst

```diff
@@ -52,17 +52,30 @@ Doing post and put requests is equally simple::
 Creating Multipart Requests with Files
 ======================================
 
-You can include files in request bodies by including them in the data array::
+You can include files in request bodies by including a filehandle in the array::
 
     $http = new Client();
     $response = $http->post('http://example.com/api', [
-      'image' => '@/path/to/a/file',
-      'logo' => $fileHandle
+      'image' => fopen('/path/to/a/file', 'r'),
     ]);
 
-By prefixing data values with ``@`` or including a filehandle in the data.  If
-a filehandle is used, the filehandle will be read until its end, it will not be
-rewound before being read.
+The filehandle will be read until its end; it will not be rewound before being read.
+
+*Warning:* For compatibility reasons, strings beginning with ``@`` will be evaluated
```

Comment:
you should use ``.. warning:`` here instead and then indent the paragraph as explain here in the [guidelines](http://book.cakephp.org/3.0/en/contributing/documentation.html#notes-and-warnings)